libraries/TGRSIAnalysis/TS3/TS3.cxx

```diff
@@ -195,26 +258,29 @@ for(size_t j = 0; j < fS3_SectorFragment.size(); ++j)  EneS.push_back(fS3_Sector
 	} //End Shared Sector loop
   }//End Shared Strip Loop
 
-
+	
   //If there are any used hit parts clear 'em
   fS3_RingFragment.clear();
   fS3_SectorFragment.clear();
 }
 
 
-
-
-
-
-
-TVector3 TS3::GetPosition(int ring, int sector)  {
+TVector3 TS3::GetPosition(int ring, int sector, bool downstream, double offset)  {
```

Comment:
I suggest that it would be helpful to include default values for downstream and offset so that any codes that have previously used the GetPosition(int,int) function work properly.